docroot/modules/humsci/hs_paragraph_types/modules/hs_row/modules/hs_row_display/hs_row_display.module

```diff
@@ -8,6 +8,20 @@
 use Drupal\Core\Entity\EntityInterface;
 use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
 
+/**
+ * Implements template_preprocess_paragraph().
+ */
+function hs_row_display_preprocess_paragraph__hs_row(&$variables) {
+  $paragraph = $variables['paragraph'];
+  $row_style = $paragraph->get('field_paragraph_style')->value;
```

Comment:
@pookmish let me know if this looks better for getting field data from an entity.

Reply:
I would still prefer `->getString()`. also this should always check if the field is there before trying to get it.